sdk/communication/tests.yml

```
@@ -16,7 +16,9 @@ stages:
       parameters:
         CloudConfig:
           Public:
-            SubscriptionConfiguration: $(sub-config-communication-services-cloud-test-resources)
+            SubscriptionConfiguration:
```

This needs to be SubscriptionConfigurations

```suggestion
            SubscriptionConfigurations:
```

---

Also, once we have all your PRs up, we should do the switch to use `sub-config-communication-services-cloud-test-resources` instead of `sub-config-communication-services-cloud-test-resources-test` before merging.